### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by chriscantrell1989** [brianpetro/obsidian-smart-connections#788] [General]
Hello!

I am going in a circle and could use some ideas. I am setting up Smart Connections for the first time
- I can chat with the smart chat AI in the side window. It cannot see my notes (See picture below
- I have input my openAI key
- I have tried toggling on/off SmartConnection (uninstall/re-install too.)

Thanks!! 

![image](https://github.com/user-attachments/assets/cc7cb1b9-597b-44dc-9b73-f48b0017d6d9)
![image](https://github.com/user-attachments/assets/82c3537f-21f5-41a2-8a97-525ae173812d)
![image](https://github.com/user-attachments/assets/68cc54d9-b46b-4324-975e-9105f4fef41e)
![image](https://github.com/user-attachments/assets/9f4c8022-b1fd-4cf5-94db-28a4e271e0a7)
![image](https://github.com/user-attachments/assets/e012ed68-a23b-4f35-81de-8a93eb7394d3)
![image](https://github.com/user-attachments/assets/fbbbfabe-ea43-484e-9725-52587209c95f)

**Comment by christianngarcia** [comment]
Try based on my notes, and then ask a question. The "my" is a self-referential pronoun that will trigger the look up. Currently this is the only way that Smart Chat is able to access your notes. 😊

**Reply by chriscantrell1989** [reply]
hi @christianngarcia - thank you for the help. Unfortunately no luck. Any ideas? I am not sure where else I can get help.

<img width="257" alt="image" src="https://github.com/user-attachments/assets/002afed7-1494-4c1e-96bb-99ec2c2317f7">

**Reply by christianngarcia** [reply]
Make sure your on the latest version. The undefined bug should be fixed.

**Comment by nicholaswgd** [comment]
Hi,

This is probably not the right place to ask this question, but if you could tell me about it, I recently opened a thread on link: https://forum.obsidian.md/t/plugin-smart-connections-how-to-connect-the-smart-connection-plugin-to-chatgpt/89833

Where basically I am having difficulties in understanding how to connect the Obdisian with a Plugin Smart Connections.

**Comment by chriscantrell1989** [comment]
Thanks for reaching out! I have given up and it is now really far out of my
mind. I appreciate you reaching out.
